snap/snapcraft.yaml

```diff
@@ -8,6 +8,14 @@ description: |
 grade: stable
 confinement: strict
 
+hooks:
+  install:
+    plugs:
+        - network
+        - network-bind
+        - home
+        - dot-kube-config
```

Comment:
**question** what are these plugs for? Do we need all of them? BTW I don't see any install hook script in the PR

Reply:
I limit the plug to `home`. The install script was already in the repository